scripts/docs-v2.sh

```diff
@@ -1,11 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
@seratch Ive removed this file since I cannot find a reference to it, but let me know if it is still used somewhere?

Reply:
@WilliamBergamin Yes, this is still used (rarely though!). The v2 document is live here: https://slack.dev/python-slackclient/ and the contents are available in this private repo: https://github.com/slackapi/slackapi.github.io/tree/master/python-slackclient There is no guide in this repo (that's not good, sorry!) but we manually copy the generated outputs to the python-slackclient directory. We can delete this operation in the future (say, a few years later) but to me it sounds a bit early to do so today.

Reply:
Ohh I did not know [this page](https://slack.dev/python-slackclient/) was still available, it makes sense to leave the docs in as long as `slackclient` is being used